src/mrpro/utils/_zero_pad_or_crop.py

```diff
@@ -75,7 +75,7 @@ def zero_pad_or_crop(
         new_shape = tuple(new_shape[dim.index(i)] if i in dim else s for i, s in enumerate(data.shape))
 
     npad = []
-    for old, new in zip(data.shape, new_shape):
+    for old, new in zip(data.shape, new_shape, strict=False):
```

Comment:
thi sshould also be strict=true imho

Reply:
changed in [c0e2d68](https://github.com/PTB-MR/mrpro/pull/193/commits/c0e2d68acdd11289bc3819520efef2bef172fa2c)